runtime/include/vendor_version.h

```diff
@@ -31,4 +35,8 @@
  *
  * #define VENDOR_SHORT_NAME "ABC"
  * #define VENDOR_SHA "1a2b3c4"
+ *
+ * Example usage for inclusion of a vendor version string.
+ * This value will be inserted into the system property java.vm.version.
```

Comment:
I don't think the word `inserted` is appropriate. java.vm.version is set to this value.

Reply:
Fixed. Pls have another look.